### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 9

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#475] [Esami & Esoneri M-Z]
### Traccia

<img width="900" alt="image" src="https://github.com/user-attachments/assets/d7e5c28d-6c99-4cdf-952a-419caba2c9e0">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/80c72622-49b8-4012-9446-90e646a0a488)

## Tableau

![image](https://github.com/user-attachments/assets/27061769-5e61-4785-ba6e-aabc7127002e)


![image](https://github.com/user-attachments/assets/2b31c8b4-da00-4512-8f29-9d8ce937d0cc)
